test/dice_test.coffee

```diff
@@ -0,0 +1,27 @@
+Helper = require('../node_modules/hubot-test-helper/src/index.coffee')
+
+helper = new Helper('../scripts/dice.coffee')
+
+expect = require('chai').expect
```

Comment:
I think that's actually already a pretty compelling case :(

![fuck js](http://pixxx.wtf.cat/image/2t0z041Y340l/Image%202015-04-08%20at%201.40.02%20PM.png)

It can stay as is :+1: